ginga/rv/main.py

```diff
@@ -323,6 +329,8 @@ def main(self, options, args):
                               font_scaling_factor=None,
                               save_layout=True,
                               use_opengl=False,
+                              layout_file='layout.json',
+                              plugin_file='plugins.json',
```

Comment:
`plugins.json` looks like a new feature but I cannot find enough documentation to understand what it does.

Reply:
It's just the list of plugins, in JSON format. If present, it will override the built in list (in `main`). Currently there is nothing in Ginga to write out this file, but that would be something to possibly add in the future (e.g. allow users to easily customize their plugins). I am going to provide 2 or 3 example configurations with plugin lists.

Reply:
> `plugins.json` looks like a new feature but I cannot find enough documentation to understand what it does.

@pllim, can you check out the new documentation in the manual page on customization? See if it explains the layout and plugins files to your satisfaction.

Reply:
So, the "standard layout" example in https://ginga--763.org.readthedocs.build/en/763/manual/customizing.html#format-of-the-layout-table is the Python one, right? Not JSON? Maybe should be clarified.

I also think example `plugin.json` should be formatted similarly to https://ginga--763.org.readthedocs.build/en/763/manual/customizing.html#customizing-the-set-of-plugins ; i.e., the line breaks. I like the extra line breaks in the doc example.

Reply:
I'm ok with that. Makes it a lot longer though.